src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -24,6 +26,7 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     UserService userService = UserServiceFactory.getUserService();
+    Map<String, Object> jsonInfo = new HashMap<>();
```

Comment:
Should move this down to where it's used and add a comment explaining why you're using a hashMap.